.travis.yml

```diff
@@ -37,15 +37,15 @@ jobs:
     - stage: deploy
       python: '3.6'
       env:
-      install: skip
+      install: pip install -U setuptools setuptools_scm
```

Comment:
Why this change? I think we don't need this because this dependencies are already configured in `setup.py`.